src/pkg/types/driver.go

```diff
@@ -21,6 +21,8 @@ type Container struct {
 	VolumesFrom []string // container (default rw), container:rw, or container:ro
 	EntryPoint  []string
 	Command     []string // overridden by Run()
+	WorkDir     *string
```

Comment:
why ptr? `""` is a good default here, no?

Reply:
So we can pass in nil to allow the default `WORKDIR` specified in the dockerfile to take effect, setting it to "" may modify the behaviour.

Reply:
just detect `""` and pass `nil`? `WorkDir: ""` makes no sense anyway

Reply:
I think using pointer probably is more idiomatic to represent an optional type due to how aws sdk use it nowadays.